assets/src/stories-editor/components/higher-order/with-amp-story-settings.js

```diff
@@ -37,7 +37,8 @@ import {
 	BLOCKS_WITH_TEXT_SETTINGS,
 	BLOCKS_WITH_COLOR_SETTINGS,
 	MIN_BLOCK_WIDTH,
-	MIN_BLOCK_HEIGHT,
+	MIN_BLOCK_HEIGHTS,
+	BLOCKS_WITHOUT_RESIZING,
```

Comment:
Minor naming nitpick: the other constants are named `BLOCKS_WITH_...` - can we use the same here? In other words, whitelist instead of blacklist.

Reply:
Changed within 90039dd.